common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/RetryingBlockTransferor.java

```diff
@@ -34,44 +34,48 @@
 import org.apache.spark.network.util.TransportConf;
 
 /**
- * Wraps another BlockFetcher with the ability to automatically retry fetches which fail due to
- * IOExceptions, which we hope are due to transient network conditions.
+ * Wraps another BlockFetcher or BlockPusher with the ability to automatically retry fetches or
+ * pushes which fail due to IOExceptions, which we hope are due to transient network conditions.
  *
- * This fetcher provides stronger guarantees regarding the parent BlockFetchingListener. In
+ * This transferor provides stronger guarantees regarding the parent BlockTransferListener. In
  * particular, the listener will be invoked exactly once per blockId, with a success or failure.
  */
-public class RetryingBlockFetcher {
+public class RetryingBlockTransferor {
 
   /**
-   * Used to initiate the first fetch for all blocks, and subsequently for retrying the fetch on any
-   * remaining blocks.
+   * Used to initiate the first fetch or push for all blocks, and subsequently for retrying the
```

Comment:
Since this class has been renamed with the general suffix - `Transferor`, I think we can just say `... fist transfer for all blocks ...` here but hide the specific transfer type.